### PR TITLE
support bySeriesWithTagsFnv, move str->method logic to schema

### PR DESCRIPTION
**Pull request by Dieterbe**
we forgot to include support for bySeriesWithTagsFnv. the functionality was in the schema package, but not yet a way to use it.

### CODE REVIEW

**Comment by Dieterbe** [comment]
note to self: once approved, do the same for tsdb-gw

**Comment by robert-milan** [comment]
> we forgot to include support for bySeriesWithTagsFnv. the functionality was in the schema package, but not yet a way to use it.

We were just waiting until we could merge the performance improvements from #1436.

File: docs/tools.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ Flags:
   -num-partitions int
     	number of partitions in cluster (default 1)
   -partition-scheme string
-    	method used for partitioning metrics. (byOrg|bySeries|bySeriesWithTags) (default "byOrg")
+    	method used for partitioning metrics. (byOrg|bySeries|bySeriesWithTags|bySeriesWithTagsFnv) (default "byOrg")
```
Comment (robert-milan) [inline docs/tools.md]: Would this be a good time to change the default to `bySeriesWithTags`?

Comment (Dieterbe) [reply docs/tools.md]: that requires a bit more coordination. i've created a separate ticket and ping you.

**Pull request merged**